2.Nomic_nELISA_Analysis/1.umap/PBMC/scripts/PBMC_umap_Visualization.r

```diff
@@ -0,0 +1,112 @@
+library(ggplot2)
+library(gridExtra)
+library(cowplot)
+
+cell_type <- "PBMC"
+
+# import paths
+
+nomic_data_path <- "./results/nELISA_plate_430420_umap_PBMC.csv"
+nomic_data_selected_treatments_path <- "./results/nELISA_plate_430420_umap_PBMC_selected_treatments.csv"
```

Comment:
Would it make sense to use `file.path()` as a rough `pathlib` equivalent to other work in this PR?

Reply:
added!